Added link to Swagger UI from keycloak Dev UI Screen

phillip-kruger opened the pull request:
Adds a link to Swagger UI from the Keycloak Dev UI, if OpenAPI is included.

We plan to play some more with this to see if that link can included the current logged in user.

![image](https://user-images.githubusercontent.com/6836179/130964448-8c8542d1-9077-4624-8021-32ac3d036df9.png)


Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

manovotn committed:
Arc - type-level and method-level inheritance tests

loicmathieu committed:
Liquibase MongoDB extension

Fixes #18009, #9801

patriot1burke committed:
aws testing endpoint

aws test event server

hot replacement lambda

hot replacement lambda cleanup

hot replacement lambda cleanup 2

lambda http finish

add gson to benchmarks

do not enable mock lambda if LambdaClint

final dev mode touches lambda

lambda native testing

comment

comment

finish it

process resources

fix banned dependency

fix banned dependency

fixes

debug lambda test mode

finish mock event initial

process res

logging remove

lambda fixes

lambda continuous testing test

stuartwdouglas committed:
Add some substitutions for Brotli and Zstd

Ladicek committed:
Simplify logging using static invocations and bytecode transformation

Also update Jandex to 2.4.0.Final and Jandex Maven plugin to 1.2.0.

sberyozkin committed:
Share containers in DevServices for Keycloak

michalszynkiewicz committed:
gRPC: simplify adding headers to client calls

fixes #19209

geoand committed:
Add the ability for extensions to exclude GraalVM config from jars

Resolves: #17482

geoand committed:
Add the ability for extensions to exclude GraalVM config from jars

Resolves: #17482

Sanne committed:
Upgrade to Hibernate ORM 5.5.7.Final

dependabot[bot] committed:
Bump awssdk.version from 2.17.26 to 2.17.27

Bumps `awssdk.version` from 2.17.26 to 2.17.27.

Updates `software.amazon.awssdk:bom` from 2.17.26 to 2.17.27
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.26...2.17.27)

Updates `apache-client` from 2.17.26 to 2.17.27

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

famod committed:
Revert "Revert "Bump to Liquibase 4.4.0""

This reverts commit bbbaf7687d5968e74fd3e68eda774cc3500dcb0f.

famod committed:
Revert "Revert "Fix broken Liquibase 4.4 in native mode""

This reverts commit 32ba42794441a29cd51d4b7a2023eadddccfbb83.

famod committed:
Bump liquibase to 4.4.3

stuartwdouglas committed:
Improve DevServices test integration

Also improce the KeyCloak test client, and modify a KC test to use
DevServices.

This adds a new DevServicesContext which can be injected into
QuarkusTest and QuarkusIntegrationTest, as well as TestResourceManager
implementation (to allow the dev service to be configured).

geoand committed:
Add `forRemoval` to RESTEasy Reactive deprecations

michalszynkiewicz committed:
Reactive Messaging WebSocket Guide

manovotn committed:
Arc - Filter out overriden methods when looking for pre-destroy and post construct candidated.

Sanne committed:
Upgrade the Gradle dependency test to match the ByteBuddy version of Hibernate ORM

geoand committed:
Add an actionable message when attribute of multipart is a file and not accessed as such

Closes: #19677

aloubyansky committed:
Fallback to the original registry repo config in case applied mirror/proxy from settings didn't work, better logs

phillip-kruger committed:
Dev UI indicate properties set by DevServices

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

geoand committed:
Remove unnecessary Jackson dependency from oidc extension

See https://github.com/quarkusio/quarkus/issues/19688#issuecomment-906358106
for the reasoning behind this

Fixes: #19688

phillip-kruger committed:
Added link to Swagger UI from keycloak Dev UI Screen
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

sberyozkin reviewed (approved):
Hi Phillip, this is a nice start, lets give it a go and see how it works out. 
Indeed passing the acquired token to it would be great.
In the context of this Keycloak Dev UI, `swagger-ui` should act as you showed the other day, where it has a field for a token

aloubyansky committed:
Check whether the platform catalog couldn't be resolved from a Maven mirror and indicate that it might not be up-to-date in the error message

gsmet committed:
Upgrade build-reporting-maven-extension to 1.0.3

sberyozkin committed:
Make it easier to access the static OidcTenantConfig from custom tenant resolvers

aloubyansky committed:
Fix dev mode launch on Windows, replace jline with Aesh in the quarkus-maven-plugin

manovotn committed:
Arc - pre-destroy and post construct shouldn't be considered as candidates for around invoke interception. Add a test for various kinds of interception together.

dependabot[bot] committed:
Bump awssdk.version from 2.17.27 to 2.17.28

Bumps `awssdk.version` from 2.17.27 to 2.17.28.

Updates `software.amazon.awssdk:bom` from 2.17.27 to 2.17.28
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.27...2.17.28)

Updates `apache-client` from 2.17.27 to 2.17.28

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump sentry-jul from 5.1.1 to 5.1.2

Bumps [sentry-jul](https://github.com/getsentry/sentry-java) from 5.1.1 to 5.1.2.
- [Release notes](https://github.com/getsentry/sentry-java/releases)
- [Changelog](https://github.com/getsentry/sentry-java/blob/main/CHANGELOG.md)
- [Commits](https://github.com/getsentry/sentry-java/compare/5.1.1...5.1.2)

---
updated-dependencies:
- dependency-name: io.sentry:sentry-jul
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump aws-lambda-java-events from 3.9.0 to 3.10.0

Bumps [aws-lambda-java-events](https://github.com/aws/aws-lambda-java-libs) from 3.9.0 to 3.10.0.
- [Release notes](https://github.com/aws/aws-lambda-java-libs/releases)
- [Commits](https://github.com/aws/aws-lambda-java-libs/commits)

---
updated-dependencies:
- dependency-name: com.amazonaws:aws-lambda-java-events
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

stuartwdouglas committed:
Handle creating the src dir in dev mode

Fixes #19655

dimas-b committed:
Add config property for reusing jacoco data files

This is to allow aggregating coverage results from
multiple test JVMs, for example when using the Surefire
property reuseForks = false

JaCoCo itself has configuration settings to allow
appending coverage data to an existing file (which is
its default behaviour).

The Quarkus default behaviour is unchanged (the jacoco
data file is deleted before each run).

stuartwdouglas committed:
Fix "Response head already sent"

Fixes #19621

gastaldi committed:
Extension registry repository as a Nexus Repository proxy

mkouba committed:
Qute - unwrap java.util.Optional automatically

- also handle empty Optional specially in the "or" resolver
- resolves #19554

snazy committed:
Fix NPE in TenantConfigBean preventing proper ARC shutdown leading to class loader issues

geoand committed:
Revert "Add the ability for extensions to exclude GraalVM config from jars"

glefloch committed:
Remove hard coded dependency version in ConditionalDependenciesTest

Sanne committed:
Upgrade to Hibernate ORM 5.6.0.Beta1

mkouba committed:
Qute type-safe validation - include all interface methods in hierarchy

- resolves #19564

mkouba committed:
ArC - use the context object for a custom pseudo-scope

- resolves #19725

paulrobinson committed:
Updated "Dev UI page" text to link to Dev UI on localhost

gastaldi committed:
Bump com.gradle.enterprise plugin to 3.6.4

According to https://docs.gradle.com/enterprise/gradle-plugin/#gradle_6_x_and_later

wojciechstryjewskipl committed:
Add audit table suffix config property

sberyozkin committed:
Document how to test OIDC with DevServices and minor updates to quarkus-test-keycloak-server

paulrobinson committed:
Small typo

joyrex2001 committed:
Fix async i/o error and move startup script

gastaldi committed:
Add Nexus 3 instructions

dependabot[bot] committed:
Bump awssdk.version from 2.17.28 to 2.17.29

Bumps `awssdk.version` from 2.17.28 to 2.17.29.

Updates `software.amazon.awssdk:bom` from 2.17.28 to 2.17.29
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.28...2.17.29)

Updates `apache-client` from 2.17.28 to 2.17.29

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump jackson-bom from 2.12.4 to 2.12.5

Bumps [jackson-bom](https://github.com/FasterXML/jackson-bom) from 2.12.4 to 2.12.5.
- [Release notes](https://github.com/FasterXML/jackson-bom/releases)
- [Commits](https://github.com/FasterXML/jackson-bom/compare/jackson-bom-2.12.4...jackson-bom-2.12.5)

---
updated-dependencies:
- dependency-name: com.fasterxml.jackson:jackson-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

asamal committed:
Update docs to remove not working examples

culmat committed:
Update vertx.adoc

fix typo

famod committed:
Fix wrong TestInfo test method in @Before/AfterEach

Fixes #19747

dufoli committed:
stereotype and qualifier inheritence and transitive stereotype in arc

geoand committed:
Provide actionable error message when non-static inner class used as JAX-RS resource

Closes: #19757

geoand committed:
Use testcontainers (if applicable) to determine if Docker working

Fixes: #19741

geoand committed:
Provide actionable error message when RESTEasy can't pick Provider constructor

This can happen when a provider is registered via a Dynamic Feature.
Unfortunately the error message can not contain the class which causes the problem
because it is not passed into the method, the class itself is not present in fields
which can be accessed at this point and the StackWalker does not have access to parameters

Closes: #19756

phillip-kruger committed:
fix issue in dev ui logstream that register listener on all select elements.
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

geoand committed:
Allow RESTEasy Reactive to index JAX-RS Resources generated by other extensions

gsmet committed:
Remove two classes that has been moved to test-framework/artemis

phillip-kruger committed:
Show Dev UI Tile even if there is no Guide.
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

geoand committed:
Use the proper config values when pulling base image for native build using Jib

Fixes: #19771

phillip-kruger committed:
Some Swagger UI Screenshots updates in the Guides
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

sberyozkin committed:
Update OIDC TokenStateManager to return Uni

gsmet committed:
Upgrade build-reporting-maven-extension to 1.0.4

Fixes module path relativization for incremental builds.

famod committed:
Add timestamp to each StepTiming message

dependabot[bot] committed:
Bump awssdk.version from 2.17.29 to 2.17.30

Bumps `awssdk.version` from 2.17.29 to 2.17.30.

Updates `software.amazon.awssdk:bom` from 2.17.29 to 2.17.30
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.29...2.17.30)

Updates `apache-client` from 2.17.29 to 2.17.30

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

stuartwdouglas committed:
Update Dev UI with the correct state when resuming

Fixes #19777

geoand committed:
Support non-parameterized List method param for Query Param

RESTEasy Classic supports this, so let's remain compatible

geoand committed:
Port rest-data-panache to RESTEasy Reactive

Resolves: #19771

glefloch committed:
Set default jvm-target in kotlin sample project

phillip-kruger committed:
OpenAPI Auto security based on extensions

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

geoand committed:
Use proper type for method comparison in REST Links module

geoand committed:
Fix issue with ResourceInfo injection in response filters

manusa committed:
fix: K8s Service can be re-applied

Signed-off-by: Marc Nuri <marc@marcnuri.com>

mkouba committed:
Codestarts - gRPC - add index.html

- resolves #19642

geoand committed:
Remove query parameters UriInfo#getAbsolutePath in RESTEasy Reactive

This is mentioned by the method's Javadoc, even though the
TCK doesn't test it

gemmellr committed:
update to proton-j 0.33.9

geoand committed:
Fix Jackson serialization issue in Panache REST Data

Before this change the custom serializer would not
take the registered Jackson modules into account

geoand committed:
Ensure that Violation DTO class is registered in native

geoand committed:
Add reflection registration for REST links module

Sanne committed:
Make sure the --exclude-config arguments precede the -jar arguments

cescoffier committed:
Update Vert.x version to 4.1.3 and Mutiny bindings to 2.13.0

famod committed:
Fix NoSuchElementException on `@Before/AfterAll` with TestInfo

Fixes #19800

ebullient committed:
CLI: docs and help for -P

glefloch committed:
Do not propagate java compiler argument in kotlin compilation provider

dependabot[bot] committed:
Bump awssdk.version from 2.17.30 to 2.17.31

Bumps `awssdk.version` from 2.17.30 to 2.17.31.

Updates `software.amazon.awssdk:bom` from 2.17.30 to 2.17.31
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.30...2.17.31)

Updates `apache-client` from 2.17.30 to 2.17.31

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

stuartwdouglas committed:
Don't rely on vert.x for CT tests

stuartwdouglas committed:
Fix some tests the broke due to default change

The max attribute size changed in vert.x

cescoffier committed:
Update the status of the mutiny, AMQP connector and gRPC extensions to "stable".

phillip-kruger committed:
Dev UI Config screen render config editor based on type (Fix #19636)
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

stuartwdouglas committed:
Make sure KC Dev UI page persists after reload

geoand committed:
Fix @NoCache without fields handling in RESTEasy Reactive

Fixes: #19822

geoand committed:
Mark quarkus-container-image-openshift as stable

loicmathieu committed:
Add missing @ConfigItem annotations

phillip-kruger committed:
Dev UI Config screen show lock icon for Build Time properties (Fix #19755)
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

michalszynkiewicz committed:
Rest Client Reactive: Multipart file name set to actual filename for path and file

fixes #19805

jmartisk committed:
SmallRye GraphQL 1.3.2

geoand committed:
Provide actionable error message when RESTEasy Reactive can't pick Provider constructor

Relates to: #19750

Co-authored-by: George Gastaldi <gegastaldi@gmail.com>

michalszynkiewicz committed:
Support config properties in @ClientHeaderParam

fixes #19805

manusa committed:
Bump kubernetes-client-bom from 5.7.0 to 5.7.2

Signed-off-by: Marc Nuri <marc@marcnuri.com>

phillip-kruger committed:
Dev UI Config screen copy-to-profile takes filter into account (Fix #19821)
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

glefloch committed:
Remove hard coded kotlin version in Gradle tests

gastaldi committed:
Marking Amazon extensions as stable

geoand committed:
Replace runtime reflection usage in rest-links with build time metadata capturing

geoand committed:
Take @TestSecurity(authorizationEnabled = false) into account for RESTEasy Reactive

Relates to: #19834

geoand committed:
Don't fail Hibernate Validator when no RESTEasy Reactive request is in flight

Relates to: #19834

phillip-kruger committed:
Dev UI Test result screen make sure dark background use light test (Fix #19374)
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

aloubyansky committed:
Removed duplicate dependency on asm-commons, trigerring a warning during the build

phillip-kruger committed:
Dev UI Keykloak - Make the logout button visible
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

michalszynkiewicz committed:
Rest Client Reactive: improve error on invalid subresource return type

dependabot[bot] committed:
Bump awssdk.version from 2.17.31 to 2.17.32

Bumps `awssdk.version` from 2.17.31 to 2.17.32.

Updates `software.amazon.awssdk:bom` from 2.17.31 to 2.17.32
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.31...2.17.32)

Updates `apache-client` from 2.17.31 to 2.17.32

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump flyway-core from 7.14.0 to 7.14.1

Bumps [flyway-core](https://github.com/flyway/flyway) from 7.14.0 to 7.14.1.
- [Release notes](https://github.com/flyway/flyway/releases)
- [Commits](https://github.com/flyway/flyway/compare/flyway-7.14.0...flyway-7.14.1)

---
updated-dependencies:
- dependency-name: org.flywaydb:flyway-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

stuartwdouglas committed:
Clean up hang detection on failed start

This can cause a false positive in the vertx-http-deployment tests where
a continuous test fails to start, then spits out a hang detection
warning a minute later.

stuartwdouglas committed:
Fix race in continuous testing tests

stuartwdouglas committed:
Change dev mode working directory

This will now be the same as surefire, so tests will see the same
user.dir for both surefire tests and continuous tests.

This is technically a breaking change, but should not really affect the
majority of applications. The old behaviour can be restored using

<workingDir>${project.build.directory}</workingDir>

in the maven plugin configuration.

aloubyansky committed:
Experimental Maven goal to check for Quarkus platform and extension updates

Co-authored-by: Guillaume Smet <guillaume.smet@gmail.com>

iocanel committed:
chore: remove useless code from KuberentesProcessor

FroMage committed:
Make sure we respect and document the MP config settings for overriding Context Propagation

iocanel committed:
fix: KubernetesClientErrorHandler always throws a RuntimeException

geoand committed:
Improve documentation for token propagation

manovotn committed:
Arc - transactional observers should register standard synchronization instead of interposed.

barreiro committed:
Expose Agroal feature: idle validation

Ladicek committed:
More descriptive error messages for unsupported Jandex storage format versions

sberyozkin committed:
Dev Services for Keycloak: read the realm name from file, fix the container lookup, create default realm optionally and follow up KeycloakTestClient updates

kdubb committed:
Update JNA to work with Apple M1

Mahaswami committed:
Remove Standalone lines should be true by default as doc says it is

kenfinnigan committed:
Fix RESTEasy classic server templating for observability

- Fixes #18251
- Add test to Micrometer integration test. Verified not a problem with RESTEasy Reactive
- Add test to OpenTelemetry integration test to very fix for RESTEasy Classic

stuartwdouglas committed:
Use the same parent CL for both test and dev

The isolated test CL does not need to access anything from the dev CL,
and it just causes leakage problems.

Fixes #19888

gastaldi committed:
Test execution message is now copy/paste friendly

The "Executing" message shown in the ArtifactLauncher implementations now allows you to copy/paste the command easily

stuartwdouglas committed:
Add QuarkusMainTest and QuarkusMainIntegrationTest

These allow you to test running a command mode application to
completion, and verify the outcome.

Also coverts the Picocli tests to use this, rather than testing using a
rest endpoint.

stuartwdouglas committed:
Make Dev Services logging nicer

This changes the mass of Dev Services logging to a single status line
per starting container. This allows the user to see that progress is
happening, without flooding the console.

If the operation fails all captured output is dumped.

Fixes #16203

Karm committed:
LambdaContainerHandlerSubstitution class must be marked final

geoand committed:
Ensure that eager security handling works in native mode

Fixes: #19878

geoand committed:
Fix use of OpenTelemetry OTLP exporter in native mode

Essentially the fact that ClientTracingFilter was using a static
field for TextMapPropagator was causing GraalVM to initialize
the entire telemetry infrastructure at build time

Fixes: #19877

mkouba committed:
ArC - make it possible to identify additional bean archives

mkouba committed:
gRPC - register additional bean archives

- an application archive that contains a MutinyBean implementor is an
additional bean archive
- resolves #19864

glefloch committed:
Fix gradle deployment classpath leak

geoand committed:
Fix issue with skip predicates in scheduled methods

Before this fix, the implementations where actually
eligible for bean removal

Fixes: #19900

mkouba committed:
Qute docs - introduce the "Qute Used as a Standalone Library" section

Sanne committed:
Upgrade to Oracle JDBC driver v 21.3.0.0 and import its BOM

Sanne committed:
Override the Oracle JDBC automatic native-image metadata and re-implement it explicitly

aloubyansky committed:
Include relevant upstream catalogs even if their versions aren't recommended for new projects

Moved io.quarkus.maven.StreamCoords to io.quarkus.registry.catalog.PlatformStreamCoords

EricWittmann committed:
Fixed a broken link in a guide

dependabot[bot] committed:
Bump awssdk.version from 2.17.32 to 2.17.33

Bumps `awssdk.version` from 2.17.32 to 2.17.33.

Updates `software.amazon.awssdk:bom` from 2.17.32 to 2.17.33
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.32...2.17.33)

Updates `apache-client` from 2.17.32 to 2.17.33

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

manaswinidas committed:
Fix broken link for config-reference

famod committed:
Fix combination of multiple `@Nested` tests with root lifecycle methods

Fixes #19843

famod committed:
Set default jvm-target in basic kotlin sample project

manaswinidas committed:
Add instructions to access App passwords page

Sanne committed:
Introduce a build item to control flag --allow-incomplete-classpath of native-image

phillip-kruger committed:
GraphQL - make sure the panache entities are updated in the index

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

sberyozkin committed:
Mark oidc-client and oidc-client-filter as stable and update descriptions for all OIDC extensions

manovotn committed:
Make TransactionSynchronizationRegistry producer @Unremovable once again

radcortez committed:
Revert "Register @ConfigMappings directly into the Config builder"

This reverts commit fd4b3028

gsmet committed:
Fix build report archive name for Initial JDK 11 build

gsmet committed:
CI - Always include a file at the root to get the full hierarchy

The upload-artifact action takes the common ancestor as the root path of
the archive which will lead to issues when we have only one file.

Always uploading a file from the root of the repository should fix this.

sii-ggrebert committed:
add mailer-deployment to bom

geoand committed:
Make labels a general container-image setting instead of Jib specific

Closes: #19903

dependabot[bot] committed:
Bump flyway-core from 7.14.1 to 7.15.0

Bumps [flyway-core](https://github.com/flyway/flyway) from 7.14.1 to 7.15.0.
- [Release notes](https://github.com/flyway/flyway/releases)
- [Commits](https://github.com/flyway/flyway/compare/flyway-7.14.1...flyway-7.15.0)

---
updated-dependencies:
- dependency-name: org.flywaydb:flyway-core
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump awssdk.version from 2.17.33 to 2.17.34

Bumps `awssdk.version` from 2.17.33 to 2.17.34.

Updates `software.amazon.awssdk:bom` from 2.17.33 to 2.17.34
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.33...2.17.34)

Updates `apache-client` from 2.17.33 to 2.17.34

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

stuartwdouglas committed:
Fix log assertions

This should not prevent Quarkus from shutting down

stuartwdouglas committed:
SCI's can't add websocket endpoints

Fixes #19781

stuartwdouglas committed:
Fix continuous testing race

Files to watch must be updated before the listeners are notified,
otherwise the test may modify the config file before the file set has
been updated, and the tests will not be run.

geoand committed:
Make @TestSecurity work correctly with unannotated JAX-RS endpoints security feature

The use of `quarkus.security.jaxrs.deny-unannotated-endpoints=true` essentially
results in the addition of a `DenyAllInterceptor` to the invocation chain
of a JAX-RS endpoint.
Because this interceptor did not take into account the `AuthorizationController`
(like the `RolesAllowedInterceptor` already does), it would result in endpoints
being secured even though security was supposed to be disabled for the specific test.

Fixes: #19896

geoand committed:
Trim container-image configuration values

Fixes: #19956

radcortez committed:
Improved test for @ConfigMapping constructor injection with override

radcortez committed:
Document Spring Boot `@ConfigProperties` limitations and recommendations

gsmet committed:
Set Kotlin JVM target to 11 for Maven

gsmet committed:
Do not set Kotlin JVM target to 1.8 for JDK 11+

phillip-kruger committed:
Upgrade to SmallRye OpenAPI 2.1.12
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

dependabot[bot] committed:
Bump awssdk.version from 2.17.34 to 2.17.35

Bumps `awssdk.version` from 2.17.34 to 2.17.35.

Updates `software.amazon.awssdk:bom` from 2.17.34 to 2.17.35
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.17.34...2.17.35)

Updates `apache-client` from 2.17.34 to 2.17.35

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
